roles/activemq/defaults/main.yml

```diff
@@ -9,4 +9,6 @@ activemq_environment:
     - -Xms128m
     - -Xmx1g
     - $ACTIVEMQ_OPTS
-activemq_version: null
+activemq_version: 5.18.6
+activemq_dependency_url: https://archive.apache.org/dist/activemq/{{ activemq_version }}/apache-activemq-{{ activemq_version }}-bin.tar.gz
+activemq_dependency_sha512_checksum_url: https://archive.apache.org/dist/activemq/{{ activemq_version }}/apache-activemq-{{ activemq_version }}-bin.tar.gz.sha512
```

Comment:
I would move this variable to the [get_url format](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/get_url_module.html#parameter-checksum) so it can easily be changed